test/QuestionAPI-test.js

```diff
@@ -106,6 +106,7 @@ describe("Question API Contract", function () {
     it("the factory should create questions", async function () {
       // create question
       await xmetric.connect(xmetricaddr1).approve(vault.address, ethers.utils.parseEther("30"));
+      await xmetric.connect(xmetricaddr2).approve(vault.address, ethers.utils.parseEther("30"));
```

Comment:
not a biggie I would jut change number to differentiate - also add a TODO to approve amount below so we can test rejection of upvote

Reply:
also line 88 and 108 are same - I think we need to fix that, that might of been my bad when I was cleaning up

Reply:
Will do